### ISSUE TITLE
Xiaomi Redmi Note 12 Pro 5G on GAMESIR X2 PRO

### ISSUE THREAD

**Issue by Mallga** [gamesir123/app#124]
I have a Xiaomi Redmi Note 12 PRO 5G, according to the official website I need for the GAMEsir x2 PRO requires USB Power Delivery 2.0 (I have 3.0) and a Mediatex Dimensity 930 chipset or higher (I have Dimensity 1080). The company's support tells me that I better buy the X2 Bluetooh but my equipment meets the requirements, has anyone tried it ?

**Comment by VitorLeoni** [comment]
Yes. For some reason X2 pro works only in randoms devices, redmi note 12 only works pluged on charge cable so the bluetooth version is a safe choise (devs don't care about this issue)

**Comment by josiaslg83** [comment]
I bought the Xiaomi and to my disappointment the x2 pro controller doesn't work. I would like greater support from gamesir.

**Comment by alexis-dm** [comment]
I'm not sure if this was the actual solution, but I installed and ran the 'Octopus - Keymapper' app, and since then, my Gamesir X2 Pro connects without any issues (the LED stays on) and seems to be functioning normally. Even pass-through charging works, albeit with a limited current of 1000-1200mA. Interestingly, it still functions as expected even after uninstalling Octopus.

Just in case it's relevant, I did update it beforehand by connecting it to a Samsung phone. However, after the update, the LED would still blink for a while before turning off when I reconnected it to the Redmi.

**Comment by josiaslg** [comment]
Here i have a Redmi Note 12. MIUI Global 14.0.9. Android version: 13 TKQ1.221114.001. 
Installed Octopus - Keymapper and still not sync. Leds turns on and off and nonthing works.
I choose this phone because the 120hz on screen. Is a shame for gamesir not give any support to the users. A licensed controller for Xbox not working.
